features/bundle/index.js

```diff
@@ -28,11 +28,21 @@ function writeOrder(features, targetDir) {
     fs.writeFileSync(path.join(targetDir, "order"), order.join("\n"));
 }
 
+function writeCategoriesOrder(categories, targetDir) {
+    fs.writeFileSync(
+        path.join(targetDir, "categories.json"),
+        JSON.stringify(categories)
+    );
+}
+
 const features = require("./package.json").polyPodFeatures;
+const categories = require("./package.json").polyPodCategories;
 
 const targetDir = path.join(__dirname, "dist");
 if (fs.existsSync(targetDir)) fs.rmSync(targetDir, { recursive: true });
 fs.mkdirSync(targetDir);
 
 for (let feature of features) packageFeature(feature, targetDir);
 writeOrder(features, targetDir);
+
+if (categories) writeCategoriesOrder(categories, targetDir);
```

Comment:
Looks good as a first approximation, but this is 1) going to generate that file every time, when possibly the original is not going to change 2) well, that file is going to be there, maybe add it to `.gitignore`
Finally, this is probably best done on the fly while bundling; or at any other time; creating it permanently via generator script is another option. Generally, separation of concerns dictate that file generation should be done separately from anyother things.

Reply:
1. This goes along with how features are build, do we rebuilt only the features that have changed?
2. The whole `dist` directory, where this file is placed, is already in gitignore.

`categories` file will be generated when features are packaged, in our workflow it is when `build.js` is executed.
Also note that `categories` will eventually replace `order` file, so it follows the same setup/logic when being created as for `order` file.